vj4/handler/contest.py

```diff
@@ -47,7 +47,7 @@ class ContestMainHandler(contest.ContestMixin, base.Handler):
                 tdocs=tdocs, tsdict=tsdict)
 
 
-@app.route('/contest/{tid:\w{24}}', 'contest_detail')
+@app.route('/c/{tid:\w{24}}', 'contest_detail')
```

Comment:
Our discussion shows that this PR is not going to change the URL schema of contest and homework, so please revert these changes (and open a new PR if you like).